src/tests/test_utils/common_test_utils/src/ov_test_utils.cpp

```diff
@@ -108,3 +114,56 @@ void check_unique_names(const std::shared_ptr<ov::Model>& f, const std::shared_p
 std::shared_ptr<ov::op::v0::Constant> create_zero_constant(const ov::element::Type_t& et, const ov::Shape& shape) {
     return ov::op::v0::Constant::create(et, shape, {0});
 }
+
+namespace ov {
+namespace test {
+namespace utils {
+
+ov::TensorVector infer_model_via_template(const std::shared_ptr<ov::Model>& model,
+                                          const ov::TensorVector& input_tensors) {
+    std::map<std::shared_ptr<ov::Node>, ov::Tensor> inputs;
+    auto params = model->inputs();
+    OPENVINO_ASSERT(params.size() == input_tensors.size());
+    for (int i = 0; i < params.size(); i++) {
+        inputs[params[i].get_node_shared_ptr()] = input_tensors[i];
+    }
+    return infer_model_via_template(model, inputs);
+}
+
+ov::TensorVector infer_model_via_template(const std::shared_ptr<ov::Model>& model,
+                                          const std::map<std::shared_ptr<ov::Node>, ov::Tensor>& inputs) {
+    ov::Core core;
```

Comment:
use `createCoreWithTemplate` instead

Reply:
createCoreWithTemplate is located in FuncTestUtils. I dont want to make commonTestUtils dependent on FuncTestUtils. That's why I duplicated this code